app/components/forms/LazyMintForm.tsx

```
@@ -0,0 +1,156 @@
+          >
+            {isProcessing ? 'Minting...' : ' Lazy Mint'}
+          </button>
+          <button
+            onClick={(e) => {
+              e.preventDefault();
+              if (props.handleToggleSwitch) {
+                props.handleToggleSwitch();
+              }
+            }}
+            className="min-w-10 rounded bg-gray-600 px-4 py-2 text-white transition duration-300 hover:bg-gray-500"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)